1451530310339	Sync.ErrorHandler	DEBUG	Flushing file log.
1451530310341	Sync.Service	DEBUG	Exception: Error: NS_ERROR_UNKNOWN_HOST (resource://services-sync/resource.js:393:10) JS Stack trace: Res_get@resource.js:413:5 < Sync11Service.prototype._fetchInfo@service.js:526:7 < sync@enginesync.js:77:9 < onNotify@service.js:1254:7 < WrappedNotify@util.js:141:15 < WrappedLock@util.js:96:9 < _lockedSync@service.js:1248:1 < sync/<@service.js:1239:7 < WrappedCatch@util.js:70:9 < sync@service.js:1227:5
1451530310342	Sync.ErrorHandler	DEBUG	Log cleanup threshold time: 1450666310342
1451530310345	Sync.ErrorHandler	DEBUG	No logs to clean up.
1451530310345	Sync.BrowserIDManager	INFO	currentAuthState returning error.login.reason.network due to previous failure
1451530310345	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1451530310345	Sync.Status	DEBUG	Status.service: error.sync.failed => error.login.failed
1451530310346	Sync.BrowserIDManager	INFO	currentAuthState returning error.login.reason.network due to previous failure
1451530310346	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1451530310346	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1451530310348	Sync.ErrorHandler	DEBUG	Log cleanup threshold time: 1450666310348
1451530310350	Sync.ErrorHandler	DEBUG	No logs to clean up.
1451530950648	Sync.Service	DEBUG	User-Agent: Iceweasel/31.8.0 FxSync/1.33.0.20150701023259.
1451530950649	Sync.Service	INFO	Starting sync at 2015-12-30 22:02:30
1451530950651	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1451530950653	Sync.Status	INFO	Resetting Status.
1451530950653	Sync.Status	DEBUG	Status.service: error.login.failed => success.status_ok
1451530950656	Sync.BrowserIDManager	INFO	Fetching assertion and token from: https://token.services.mozilla.com/1.0/sync/1.5
1451530950656	FirefoxAccounts	DEBUG	already verified
1451530950658	Sync.BrowserIDManager	DEBUG	Getting an assertion
1451530950658	FirefoxAccounts	DEBUG	enter getAssertion()
1451530950663	FirefoxAccounts	DEBUG	getKeyPair: already have a keyPair
1451530950664	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1451530950678	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 620"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:620 < _onComplete()@resource://services-common/hawkclient.js:156 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950680	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 636"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:636 < _onComplete()@resource://services-common/hawkclient.js:159 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950680	FirefoxAccounts	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1451530950682	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 670"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:670 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:247 < _onComplete()@resource://services-common/hawkclient.js:166 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950684	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 670"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:670 < this.HawkClient.prototype._maybeNotifyBackoff()@resource://services-common/hawkclient.js:247 < _onComplete()@resource://services-common/hawkclient.js:167 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950685	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status text:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.statusText :: line 636"  data: no] Stack trace: RESTResponse.prototype.statusText()@resource://services-common/rest.js:636 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:71 < _onComplete()@resource://services-common/hawkclient.js:172 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950687	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 620"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:620 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:72 < _onComplete()@resource://services-common/hawkclient.js:172 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950689	Sync.RESTResponse	DEBUG	Caught exception fetching HTTP status code:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.status :: line 620"  data: no] Stack trace: RESTResponse.prototype.status()@resource://services-common/rest.js:620 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:73 < _onComplete()@resource://services-common/hawkclient.js:172 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950690	Sync.RESTResponse	DEBUG	Caught exception processing response headers:[Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: RESTResponse.prototype.headers :: line 670"  data: no] Stack trace: RESTResponse.prototype.headers()@resource://services-common/rest.js:670 < this.HawkClient.prototype._constructError()@resource://services-common/hawkclient.js:75 < _onComplete()@resource://services-common/hawkclient.js:172 < onComplete()@resource://services-common/hawkclient.js:213 < onStopRequest()@resource://services-common/rest.js:444 < waitForSyncCallback()@resource://services-common/async.js:99 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:141 < this.BrowserIDManager.prototype._getAuthenticationHeader()@resource://gre/modules/services-sync/browserid_identity.js:571 < Res__createRequest()@resource://services-sync/resource.js:168 < _doRequest()@resource://services-sync/resource.js:193 < Res__request()@resource://services-sync/resource.js:388 < Res_get()@resource://services-sync/resource.js:413 < Sync11Service.prototype._fetchInfo()@resource://gre/modules/services-sync/service.js:526 < sync()@resource://services-sync/stages/enginesync.js:77 < onNotify()@resource://gre/modules/services-sync/service.js:1254 < WrappedNotify()@resource://services-sync/util.js:141 < WrappedLock()@resource://services-sync/util.js:96 < _lockedSync()@resource://gre/modules/services-sync/service.js:1248 < sync/<()@resource://gre/modules/services-sync/service.js:1239 < WrappedCatch()@resource://services-sync/util.js:70 < sync()@resource://gre/modules/services-sync/service.js:1227 < <file:unknown>
1451530950691	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"message":null,"code":null,"errno":null}
1451530950691	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"message":null,"code":null,"errno":null}
1451530950692	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: null
1451530950692	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1451530950692	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1451530950693	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1451530950693	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms.
